commit be9ad087cc97e02e3d356d4b868c94adf56bd327 (refs/changes/14/3314/3)
Author: Jerry Jelinek <jerry.jelinek@joyent.com>
Date:   2018-02-08T18:53:39+00:00 (1 year, 8 months ago)
    
    OS-6592 illumos#9033 breaks SmartOS build
    Reviewed by: Dan McDonald <danmcd@joyent.com>
    Reviewed by: Jason King <jason.king@joyent.com>
    Approved by: Jason King <jason.king@joyent.com>

diff --git a/usr/src/tools/scripts/nightly.sh b/usr/src/tools/scripts/nightly.sh
index 906c2b6e92..91168264d2 100644
--- a/usr/src/tools/scripts/nightly.sh
+++ b/usr/src/tools/scripts/nightly.sh
@@ -25,7 +25,7 @@
 # Copyright 2008, 2010, Richard Lowe
 # Copyright 2011 Nexenta Systems, Inc.  All rights reserved.
 # Copyright 2012 Joshua M. Clulow <josh@sysmgr.org>
-# Copyright 2013 (c) Joyent, Inc.  All rights reserved.
+# Copyright 2018 (c) Joyent, Inc.  All rights reserved.
 # Copyright (c) 2017 by Delphix. All rights reserved.
 #
 # Based on the nightly script from the integration folks,
@@ -1716,9 +1716,9 @@ if [[ "$t_FLAG" = "y" ]]; then
 		GENOFFSETS=${TOOLS_PROTO}/opt/onbld/bin/genoffsets
 		export GENOFFSETS
 
-		CTFCONVERT=${TOOLS_PROTO}/opt/onbld/bin/${MACH}/ctfconvert
+		CTFCONVERT=${TOOLS_PROTO}/opt/onbld/bin/${MACH}/ctfconvert-altexec
 		export CTFCONVERT
-		CTFMERGE=${TOOLS_PROTO}/opt/onbld/bin/${MACH}/ctfmerge
+		CTFMERGE=${TOOLS_PROTO}/opt/onbld/bin/${MACH}/ctfmerge-altexec
 		export CTFMERGE
 
 		PATH="${TOOLS_PROTO}/opt/onbld/bin/${MACH}:${PATH}"
